testdata/dnn/onnx/generate_onnx_models.py

```diff
@@ -680,3 +681,29 @@ def forward(self, x):
 input = Variable(torch.randn(seq_len, batch, features))
 lstm = LSTM(features, hidden, batch, bidirectional=True)
 save_data_and_model("lstm_bidirectional", input, lstm)
+
+from torchvision.ops.misc import *
+n = 3
```

Comment:
unused variable?

Reply:
No, used in the next line. 
`model = FrozenBatchNorm2d(n)`

Reply:
Sorry, my mistake!